google/resource_compute_firewall_test.go

```diff
@@ -362,6 +362,8 @@ func testAccComputeFirewall_basic(network, firewall string) string {
 	return fmt.Sprintf(`
 	resource "google_compute_network" "foobar" {
 		name = "%s"
+		auto_create_subnetworks = false
```

Comment:
I am not sure I follow here... `auto_create_subnetworks` was defaulting to `true`. Why are we changing it to false now?

Reply:
auto_create_subnetworks was, pre-1.0.0, defaulting to false. As part of 1.0.0, we made it default to true.

To avoid breaking that behaviour outside a major release, we want it to continue to default to true.

In our tests, however, we want to be sure to actually use the legacy VPC API. Because auto_create_subnetworks defaults to true, we need to explicitly set it to false to be able to use ipv4_range, because you can't use ipv4_range while auto_create_subnetworks is true.